plugin.yaml

```diff
@@ -29,7 +29,7 @@ node_types:
         default: {}
     interfaces:
       cloudify.interfaces.lifecycle:
-        create: openstack.nova_plugin.server.create
+        configure: openstack.nova_plugin.server.create
```

Comment:
i'm not entirely sure this should indeed be modeled like this - mapping "configure" to a "create" operation might be a hint something's wrong :)
perhaps this could stay under "create" but have the server's "create" operation look for a relationship to a bootable volume, without actually running a relationship operation - then "preconfigure" wont be needed